cla_public/apps/base/views.py

```diff
@@ -193,33 +193,10 @@ def get_started():
         return redirect(url_for('contact.get_in_touch'))
     return redirect(url_for('scope.diagnosis'))
 
-
-def is_safe_url(url):
-    ref_url = urlparse(request.host_url)
-    test_url = urlparse(urljoin(request.host_url, url))
-    return test_url.scheme in ('http', 'https') and \
-        ref_url.netloc == test_url.netloc
-
-
-def next_url():
```

Comment:
There is another `next_url` defined in another view:
https://github.com/ministryofjustice/cla_public/blob/06ceb9dcc10041e040c75ac854d098f878281a9d/cla_public/libs/views.py#L204-L207

Moving this to `utils` could cause some (unlikely) unintended clashes with imports.

What do you think about renaming it to something more specific?